src/xdp/xsk.c

```diff
@@ -141,11 +143,27 @@ typedef struct _XSK_TX_XDP {
 } XSK_TX_XDP;
 
 typedef struct _XSK_TX {
+    //
+    // Data path fields.
+    //
     XSK_KERNEL_RING Ring;
     XSK_KERNEL_RING CompletionRing;
     UMEM_BOUNCE Bounce;
     XSK_TX_XDP Xdp;
+    union {
+        struct {
+            UINT8 Checksum : 1;
+        };
+        UINT8 Value;
+    } OffloadFlags;
+    UINT16 LayoutExtensionOffset;
+    UINT16 ChecksumExtensionOffset;
```

Comment:
Didn't you represent these numbers with fewer bits elsewhere? If so, why the difference?

Reply:
I think these offsets are consistently 16 bits.